src/panels/lovelace/cards/hui-stack-card.ts

```diff
@@ -39,6 +50,7 @@ export abstract class HuiStackCard extends LitElement implements LovelaceCard {
       const element = this._createCardElement(card) as LovelaceCard;
       return element;
     });
+    this.requestUpdate();
```

Comment:
This is because `_cards` is not a property? Why not make it a property?

Reply:
Correct. A property was my first idea too, and I have absolutely no idea why - but it doesn't work.